src/main/deploy/constants/comp/DrivetrainConstants.json

```diff
@@ -24,6 +24,9 @@
   "driveCrossTrackKp": 5.0,
   "driveCrossTrackKi": 0.5,
   "driveCrossTrackKd": 0.0,
+  "driveCrossTrackOtherCameraKp": 5.0,
+  "driveCrossTrackOtherCameraKi": 0.5,
+  "driveCrossTrackOtherCameraKd": 0.0,
```

Comment:
These gains are almost certainly wrong - making a note here so we tune this in shop before merging